olive/strategy/search_parameter.py

```diff
@@ -66,6 +69,33 @@ def to_json(self):
         return {"olive_parameter_type": "SearchParameter", "type": "Categorical", "support": self.support}
 
 
+class CategoricalValue(Categorical):
```

Comment:
why is this required? It looks to be the same as a categorical.

Reply:
Answered [here](https://github.com/microsoft/Olive/pull/1407#discussion_r1793011403)